tests/projects/utils.py

```diff
@@ -71,9 +71,8 @@ def docker_example_base_image():
                 path=cwd,
             )
         except BuildError as build_error:
-            for chunk in build_error.build_log:
-                print(chunk)
+            for _ in build_error.build_log:
+                pass
             raise build_error
         except APIError as api_error:
-            print(api_error.explanation)
```

Comment:
Same here: https://github.com/mlflow/mlflow/pull/5581/files#r841447393

Reply:
Keep print or use logger? thinking

Reply:
It looks like we don't need to print this out. The `APIError` implementation looks like this:

https://github.com/docker/docker-py/blob/a9748a8b702a3c75b46ba8c8d0490e4b8ec5ab04/docker/errors.py#L34

```python
class APIError(requests.exceptions.HTTPError, DockerException):
    """
    An HTTP error from the API.
    """
    def __init__(self, message, response=None, explanation=None):
        # requests 1.2 supports response as a keyword argument, but
        # requests 1.1 doesn't
        super(APIError, self).__init__(message)
        self.response = response
        self.explanation = explanation

    def __str__(self):
        message = super(APIError, self).__str__()

        if self.is_client_error():
            message = '{0} Client Error for {1}: {2}'.format(
                self.response.status_code, self.response.url,
                self.response.reason)

        elif self.is_server_error():
            message = '{0} Server Error for {1}: {2}'.format(
                self.response.status_code, self.response.url,
                self.response.reason)

        if self.explanation:
            message = '{0} ("{1}")'.format(message, self.explanation)

        return message
```

As you can see, ` self.explanation` is included in the error message.

Reply:
So we shouldn't do the print here

Reply:
Yep :)

Reply:
Great! :woman_dancing: